### PR TITLE
feat: Add Saved Metrics tab to metrics popover

### PR DESCRIPTION
**Pull request by kgabryje**
### SUMMARY
This PR adds a possibility of selecting saved metrics (predefined in dataset). Also fixes bugs mentioned in points 1., 2. and 4. in this comment: https://github.com/apache/incubator-superset/pull/12095#issuecomment-747936902

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
image

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

CC: @junlincc @villebro @ktmud
@adam-stasiak Can you help with testing please?

### CODE REVIEW

File: superset-frontend/src/explore/components/AdhocMetricEditPopover.jsx [hunk 1]
```diff
@@ -276,8 +335,28 @@ export default class AdhocMetricEditPopover extends React.Component {
               </Select>
             </FormGroup>
           </Tabs.TabPane>
+          <Tabs.TabPane key={SAVED_TAB_KEY} tab="Saved">
+            <FormGroup>
+              <FormLabel>
+                <strong>Saved metric</strong>
```
Comment (rusackas) [inline superset-frontend/src/explore/components/AdhocMetricEditPopover.jsx:341]:
```suggestion
                <strong>t(Saved metric)</strong>
```

Comment (kgabryje) [reply superset-frontend/src/explore/components/AdhocMetricEditPopover.jsx]: Done. I also added translations to other untranslated strings that hung around

**Comment by junlincc** [comment]
Thank you so much for this PR! it's looking great! 

minor issue: overflow when resizing
Screen Shot 2020-12-18 at 10 11 11 AM

not related to this PR - the styling of the boxes look very different, size and color.. since we are pausing the remaining refractor work until 1.0 is hardened, i wonder if we can style the old ones to match the new antd style? 
 
Screen Shot 2020-12-18 at 10 20 00 AM

other than that, it works and look good

@rusackas @kgabryje

**Review by junlincc — COMMENTED** [review]
Thank you so much for this PR! it's looking great! 

minor issue: overflow when resizing
Screen Shot 2020-12-18 at 10 11 11 AM

@rusackas is looking at it now

not related to this PR - the styling of the boxes look very different, size, corner radius and color.. since we are pausing the remaining refractor work until 1.0 is hardened, i wonder if we can style the old ones to match the new antd style? 
 
Screen Shot 2020-12-18 at 10 20 00 AM

other than that, it works and looks good

@rusackas @kgabryje

**Comment by rusackas** [comment]
@kgabryje I'll provide a PR to fix that little label overflow.

**Review by rusackas — APPROVED** [review]
LGTM! Will follow up separately on little cosmetic issues.

**Review by junlincc — APPROVED** [review]

**Pull request merged**